src/SPC/store/Config.php

```diff
@@ -137,7 +137,7 @@ public static function getExtTarget(string $name): ?array
         if (!isset(self::$ext[$name])) {
             throw new WrongUsageException('ext [' . $name . '] is not supported yet');
         }
-        return self::$ext[$name]['target'] ?? ['static'];
+        return self::$ext[$name]['target'] ?? ['static', 'shared'];
```

Comment:
this makes every extension buildable as shared extension by default. We'll have to write a test suite for shared extensions then

Reply:
What if we just add existing `ext-tests/xxx.php` for additional shared extension test? I randomly tested several extensions and they works quite well.

Reply:
Yeah, I wouldn't suspect any extensions to have an issue with this as php is generally much more supportive of glibc linux with shared linking, but we should still make sure.

We don't need to add tests under ext-tests I think. but the test-extensions.php should also test building shared extensions.